### PR TITLE
Release 2.3.0

### PR DESCRIPTION
**Pull request by MrPetovan**
Thanks @tobiasd for the translation update!

### CODE REVIEW

**Comment by tobiasd** [comment]
You have this already in production and it is working well?

**Comment by MrPetovan** [comment]
Thanks for the reminder, I took every step but the last (running `composer install`) on https://dir.friendica.mrpetovan.com and I found a bug related to the move from `container-interop/container-interop` to `psr/container`.

**Comment by tobiasd** [comment]
So, it is working now as expected?

**Comment by MrPetovan** [comment]
Now, yes.

File: src/classes/Controllers/Web/Servers.php [hunk 1]
```diff
@@ -46,7 +46,7 @@ public function render(Request $request, Response $response, array $args): array
 	{
 		$stable_version = $this->simplecache->get('stable_version');
 		if (!$stable_version) {
-			$stable_version = trim(file_get_contents('https://git.friendi.ca/friendica/friendica/raw/branch/master/VERSION'));
+			$stable_version = trim(file_get_contents('https://git.friendi.ca/friendica/friendica/raw/branch/stable/VERSION'));
```
Comment (tobiasd) [inline src/classes/Controllers/Web/Servers.php:49]: Should I create a stable branch, branced off from the current master branch, for this to work already or is this only preparation?

Comment (MrPetovan) [reply src/classes/Controllers/Web/Servers.php:49]: The stable branch forked from master already exists on my local fork, it just needs to be created on the main repo

Comment (tobiasd) [reply src/classes/Controllers/Web/Servers.php:49]: But it should be pushed to the repositories so that the directory can find the information right?

Comment (MrPetovan) [reply src/classes/Controllers/Web/Servers.php:49]: Yes.

Comment (MrPetovan) [reply src/classes/Controllers/Web/Servers.php:49]: Never mind, it already exists: https://github.com/friendica/friendica-directory/tree/stable

I thought you checked and that I just created it on my own fork.

Comment (tobiasd) [reply src/classes/Controllers/Web/Servers.php:49]: But the code references the Friendica repository, not the directory repository.

Comment (MrPetovan) [reply src/classes/Controllers/Web/Servers.php:49]: Right, sorry I misread, let me fork it from master. Aaand done.

**Comment by AlfredSK** [comment]
So... to run the released version I need to git-checkout the stable branch, right?

**Comment by MrPetovan** [comment]
Yes, although I'll bring `master` to the same release commit one last time.

**Pull request merged**